### ISSUE TITLE
Bug(or Error) about 'sorted_test_index' ？

### ISSUE THREAD

**Issue by wlhgtc** [rusty1s/pytorch_geometric#550]
According to your code:
https://github.com/rusty1s/pytorch_geometric/blob/master/torch_geometric/read/planetoid.py#L41-L43, you resort the test node(feature and label/x and y).

E.g (Use Cora Dataset for describe):`test_index[0]=2692` and `sorted_test_index[0]=1708`.
After rerank , `x_{sorted}[1708]` is actually `x_{origin}[2692]`.
Suppose there is an edge from x[0] to x_{origin}[1708], you change it to x_{origin}[2692].
It means  that you change the original, cause the edge in graph doesn't change. 
@rusty1s  hope you could help me!

**Comment by rusty1s** [comment]
This is intended, see the official code from Thomas Kipf [here](https://github.com/tkipf/gcn/blob/master/gcn/utils.py#L24-L90). Both functions yield the exact same output.

**Comment by wlhgtc** [comment]
@rusty1s  thx for your reply.
Maybe you could help me figure out another question?

If I want to use part of the origin train nodes, such all nodes that `index%5 == 0`. 
All I need to do is change their `train_mask` to `1` without resort them, am I right ?

**Comment by rusty1s** [comment]
Yes, you are free to change the `{}_mask` as you like. No need to consider any form of resorting.

**Comment by wlhgtc** [comment]
Your reply do solve my question, I will close this issue!
